rnaseq/kallisto.cwl

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Kallisto: Quant"
+baseCommand: ["/usr/bin/kallisto"] 
+requirements:
+    - class: ShellCommandRequirement
```

Comment:
Is this really using the `ShellCommandRequirement`? It should only be included if the command must be run through a shell instead of directly `exec`ed.

Reply:
it is not - removed.